src/main/cpp/main.cpp

```diff
@@ -161,11 +161,14 @@ int main(int argc, char* argv[]) {
   } else if (result["codec"].as<std::string>() == "qoi") {
     encoder.reset(new libench::QOIEncoder());
     decoder.reset(new libench::QOIDecoder());
-  } else if (result["codec"].as<std::string>() == "jxl") {
+  } else if (result["codec"].as<std::string>() == "jxl_e3") {
+    encoder.reset(new libench::JXLEncoder<3>());
+    decoder.reset(new libench::JXLDecoder());
+  } else if (result["codec"].as<std::string>() == "jxl_e2") {
     encoder.reset(new libench::JXLEncoder<2>());
     decoder.reset(new libench::JXLDecoder());
-  } else if (result["codec"].as<std::string>() == "jxl_0") {
-    encoder.reset(new libench::JXLEncoder<0>());
+  } else if (result["codec"].as<std::string>() == "jxl_e1") {
```

Comment:
```suggestion
  } else if (result["codec"].as<std::string>() == "jxl") {
```

... since this triggers the fast path.